Terminal.Gui/Views/Scroll/ScrollBar.cs

```diff
@@ -94,38 +94,38 @@ public int Position
     /// </summary>
     public event EventHandler<CancelEventArgs<int>>? PositionChanging;
 
-    /// <summary>Gets or sets the visibility for the vertical or horizontal scroll indicator.</summary>
-    /// <value><c>true</c> if show vertical or horizontal scroll indicator; otherwise, <c>false</c>.</value>
-    public bool ShowScrollIndicator
```

Comment:
Please let me know if you really don't understand what this property is. Thanks.